### DISCUSSION TITLE
Remapping buttons, when using Onion launcher?

### DISCUSSION THREAD

**Discussion by ZacharyRanes** [OnionUI/Onion#557] [Q&A]
When using Onion without the Onion launcher I can use the remap option in settings and works fine, but when I turn on Onion launcher the button mapping all reverts to what it was default. Is there a file somewhere that I can edit to change the button mapping. I know I can do stuff in retroarch but I was figure there must be another place to edit stuff.

**Comment by Wapp92** [comment]
I would like to know too. I stop using onion launcher because of this.
